examples/react/js/app.jsx

```diff
@@ -170,7 +170,7 @@ var app = app || {};
 	var model = new app.TodoModel('react-todos');
 
 	function render() {
-		React.render(
+		ReactDOM.render(
 			<TodoApp model={model}/>,
```

Comment:
This is the only code change, since this was written the 'render' method was refactored off into ReactDOM.